WBooks/TableBooks/TableBookController.swift

```diff
@@ -38,6 +39,18 @@ final class TableBookController: UITableViewController {
         tableBookView.booksTable.register(nib, forCellReuseIdentifier: CellBookView.identifier)
     }
     
+    func loadBooks() {
+        tableBookViewModel.fetchBook(onSuccess: self.reloadTable, onError: self.showError)
+    }
+    
+    func reloadTable() {
+        tableBookView.booksTable.reloadData()
+    }
+    
+    func showError(error: Error) {
+//        BUSCAR FORMATO ERROR
```

Comment:
In this case, you can show an alert with [UIAlertController](https://developer.apple.com/documentation/uikit/uialertcontroller).